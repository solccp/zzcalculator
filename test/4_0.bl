84 85
72 71
63 64
29 30
27 23
60 19
98 15
101 11
3 4
1 6
115 116
107 108
94 96
26 21
23 20
27 31
33 32
38 55
50 56
51 137
