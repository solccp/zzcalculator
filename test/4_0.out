 267227532 + 2389804604 x + 9865265424 x^2 + 24946185302 x^3 + 43244524658 x^4 + 54500920980 x^5 + 51684194860 x^6 + 37663193263 x^7 + 21358371744 x^8 + 9490314264 x^9 + 3312396458 x^10 + 907310265 x^11 + 194327374 x^12 + 32357890 x^13 + 4158672 x^14 + 408889 x^15 + 30314 x^16 + 1635 x^17 + 58 x^18 + 1 x^19
 total: 259860994187
